tests/model_serving/model_server/conftest.py

```diff
@@ -573,28 +573,33 @@ def unprivileged_s3_caikit_serverless_inference_service(
 
 @pytest.fixture(scope="package")
 def fail_if_missing_dependent_operators(admin_client: DynamicClient) -> None:
-    missing_operators: list[str] = []
-    csvs = list(
-        ClusterServiceVersion.get(
-            dyn_client=admin_client,
-            namespace=py_config["applications_namespace"],
-        )
-    )
-
-    for operator_name in py_config.get("dependent_operators", []).split(","):
-        LOGGER.info(f"Verifying if {operator_name} is installed")
-        for csv in csvs:
-            if csv.name.startswith(operator_name):
-                if csv.status == csv.Status.SUCCEEDED:
-                    break
-
-                else:
-                    missing_operators.append(
-                        f"Operator {operator_name} is installed but CSV is not in {csv.Status.SUCCEEDED} state"
-                    )
-
-        else:
-            missing_operators.append(f"{operator_name} is not installed")
-
-    if missing_operators:
-        pytest.fail(str(missing_operators))
+    if dependent_operators := py_config.get("dependent_operators"):
```

Comment:
This fixture is supposed to be added only for serverless. https://github.com/opendatahub-io/opendatahub-tests/blob/main/conftest.py#L233. May be fix it there, so that it is not inserted there for not serverless upgrade cases?

Reply:
No this is for all, raw and serverless. I have tested it